pkg/pillar/types/global.go

```diff
@@ -705,6 +707,7 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddIntItem(NetworkTestTimeout, 15, 0, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(NetworkSendTimeout, 120, 0, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(Dom0MinDiskUsagePercent, 20, 20, 0xFFFFFFFF)
+	configItemSpecMap.AddIntItem(AppContainerStatsInterval, 300, 60, 0xFFFFFFFF)
```

Comment:
That's a min of 60 seconds, and I suspect there are sitations where we'd want 5 seconds to be able to demo this better. So the min can be 1 second for all I know.